common4j/src/main/com/microsoft/identity/common/java/authorities/AzureActiveDirectoryAuthority.java

```diff
@@ -189,6 +191,11 @@ public synchronized boolean isSameCloudAsAuthority(@NonNull final AzureActiveDir
             return false;
         }
 
+        if (cloudOfThisAuthority != null && cloudOfAuthorityToCheck != null) {
+            SpanExtension.current().setAttribute(AttributeName.home_cloud_name.name(), cloudOfAuthorityToCheck.getPreferredCacheHostName());
```

Comment:
This is utility method to compare two authorities and can potentially be used to compare authorities other than Prts home cloud. Which may give wrong information in our telemetry.
Should we add this telemetry at the source where we are calling this method.

Reply:
I think the method today is also being invoked in BrokerControllerFactory

Reply:
Yes, the purpose of adding this telemetry is to see if BrokerControllerFactory's methods to get controllers for silent and interactive calls return BrokerLocalController or BrokerSSOController. 
In BrokerControllerFactory, we are invoking this method with a PRT's authority only.